spot-webdriver/user/spot-user.js

```diff
@@ -105,13 +105,38 @@ class SpotUser {
      *
      * @returns {void}
      */
-    stopP2P() {
-        // to be implemented
+    stopP2PConnection() {
         this.driver.execute(rcsName => {
-            window.spot[rcsName]._p2pSignaling.stop();
+            try {
+                window.spot[rcsName]._p2pSignaling.stop();
+            } catch (e) {
+                // Error means p2p is already destroyed.
+            }
         }, this._internalRemoteControlServiceName);
     }
 
+    /**
+     * Stops messages from being sent to or received from the signaling service.
+     *
+     * @returns {void}
+     */
+    stopSignalingConnection() {
+        this.driver.setNetworkConditions(_OFFLINE_NETWORK_CONDITIONS);
```

Comment:
I would rather leave this call in `setNetworkOffline`, as this method might be used without turning the network offline